pom.xml

```diff
@@ -47,7 +47,7 @@
     <maven.exec.plugin.version>1.2.1</maven.exec.plugin.version>
     <clean-maven-plugin-version>2.4.1</clean-maven-plugin-version>
     <frontend-maven-plugin-version>1.6</frontend-maven-plugin-version>
-    <node.version>v6.11.5</node.version>
+    <node.version>v8.9.1</node.version>
```

Comment:
shh, don't tell anyone I snuck this in already :-)

Reply:
don't think you can.. it breaks circleci .  see this build I had to cancel:
https://circleci.com/gh/syndesisio/syndesis/343

Reply:
Strange, this one worked -> https://circleci.com/gh/syndesisio/syndesis/340?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link